GMnetENGINE.gmx/scripts/htme_clientStart.gml

```diff
@@ -37,6 +37,31 @@ if( self.socketOrServer<0 ){
     return false;
 }
 
+// Get server port from list

```

Comment:
This does not seem right here.
Shouldn't this instead be added to obj_udphphtme_lobby and the manual?
Have a look at line 29 of obj_udphphtme_lobby's event "KEY PRESS 1". We can just give the port there from the lobby entry instead of giving the port 0 to this script.